tests/Integration/Requests/ExecuteTest.php

```diff
@@ -13,6 +13,8 @@
 
 namespace Tarantool\Client\Tests\Integration\Requests;
 
+require __DIR__.'/../../../examples/bootstrap.php';
```

Comment:
This `require` is not needed. There are already two ways to detect Tarantool version in tests:
 * via `@requires Tarantool <version_constraint>` annotation ([example](https://github.com/tarantool-php/client/blob/31c3890f149c0b1b6dc7d505fbc6a2d6a4472034/tests/Integration/Requests/PrepareTest.php#L21));
 * by calling `tarantoolVersionSatisfies(<version_constraint>)` helper method ([example](https://github.com/tarantool-php/client/blob/31c3890f149c0b1b6dc7d505fbc6a2d6a4472034/tests/Integration/Requests/PrepareTest.php#LL38C31-L38C93)).

With the first option we can create separate tests for `<2.11` and `>=2.11`, but probably a simpler option would be to use the helper method and dynamically build query based on the current Tarantool version. See my suggestion below on how this could be implemented.

Reply:
I've seen the first one, but decided that separate test cases will be not the best idea. I had missed the second one. Now all tests use `tarantoolVersionSatisfies`, thank you for pointing out!